NitroxClient/GameLogic/InitialSync/VehicleInitialSyncProcessor.cs

```diff
@@ -14,15 +15,18 @@ public VehicleInitialSyncProcessor(Vehicles vehicles)
             this.vehicles = vehicles;
 
             DependentProcessors.Add(typeof(BuildingInitialSyncProcessor));
+            DependentProcessors.Add(typeof(CyclopsInitialAsyncProcessor));
```

Comment:
Do we also need to add one to BuildingInitialSyncProcessor as the Cyclops can contain furniture?

Reply:
We can set that for the future but for now its irrelevant as furniture will never stay in a cyclops when placed by another player (open ToDo)

Reply:
I added it, just to be sure